vcluster/_partials/deploy/install-beta-cli.mdx

```diff
@@ -27,28 +27,28 @@ import TabItem from "@theme/TabItem";
 <TabItem value="mac">
   
   ```bash
-  curl -L -o vcluster "https://github.com/loft-sh/vcluster/releases/download/v0.20.0-beta.6/vcluster-darwin-amd64" && sudo install -c -m 0755 vcluster /usr/local/bin && rm -f vcluster
+  curl -L -o vcluster "https://github.com/loft-sh/vcluster/releases/download/v0.20.0-beta.9/vcluster-darwin-amd64" && sudo install -c -m 0755 vcluster /usr/local/bin && rm -f vcluster
```

Comment:
This is frequent are that needs change. Would it be better to use a placeholder like `download/<VCLUSTER_VERSION>/vcluster-darwin-amd64`?

Alternatively, is it possible to keep the side wide variables external to the mdx. files, to that we only have to change place?

Reply:
We specifically didn't want to put in a random place holder so that people could copy and paste.